packages/signature-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
+### Changed
+- **BREAKING:** The constructor option `getCurrentChainId` now expects a `Hex` return value rather than `string` ([#1367](https://github.com/MetaMask/core/pull/1367))
+- Add `@metamask/utils` dependency ([#1367](https://github.com/MetaMask/core/pull/1367))
+- Support MMI use cases in signing typed messages ([#1364](https://github.com/MetaMask/core/pull/1364))
```

Comment:
Should we copy the corresponding changelog entry you left for `@metamask/message-manager`?

``` suggestion
- Emit the event `${methodName}:signed` when the keying is asked to sign a message ([#1364](https://github.com/MetaMask/core/pull/1364))
```

Reply:
Addressed here: 4b30357adee6c2ec29d8c411bffa02f2c653244c